contracts/interfaces/IBond.sol

```diff
@@ -162,11 +162,11 @@ interface IBond {
         @notice This one-time setup initiated by the BondFactory initializes the
             Bond with the given configuration.
         @dev New Bond contract deployed via clone. See `BondFactory`.
-        @dev Not calling __AccessControl_init or __ERC20Burnable_init here since
-            they currently generate an empty function.
+        @dev During initialization, __Ownable_init is not called because the
```

Comment:
[__Ownable_init](https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/cf68a02973de4a8745dc457a82d48ce238419980/contracts/access/OwnableUpgradeable.sol#L33)

If we called ownable_init() I believe it would transfer the ownership to the bondFactory contract, not the users EOA

Reply:
good point. updated wording